src/API/Management/Connections.php

```diff
@@ -1,122 +1,187 @@
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/delete_connections_by_id
      */
-    public function delete($id) 
+    public function delete($id)
     {
-        return $this->apiClient->delete()
-            ->connections($id)
+        return $this->apiClient->method('delete')
+            ->addPath('connections', $id)
             ->call();
     }
 
     /**
-     * @param string $id
-     * @param string $email
-     * @return mixed
+     * Delete a specific User for a Connection.
+     *
+     * @param string $id - Connection ID (currently only database connections are supported).
+     * @param string $email - Email of the user to delete.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/delete_users_by_email
      */
-    public function deleteUser($id, $email) 
+    public function deleteUser($id, $email)
     {
-        return $this->apiClient->delete()
-            ->connections($id)
-            ->users()
+        return $this->apiClient->method('delete')
+            ->addPath('connections', $id)
+            ->addPath('users')
             ->withParam('email', $email)
             ->call();
     }
 
     /**
-     * @param array $data
-     * @return mixed
+     * Create a new Connection.
+     *
+     * @param array $data - Connection create data; "name" and "strategy" fields are required.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/post_connections
      */
-    public function create($data) 
+    public function create($data)
     {
-        return $this->apiClient->post()
-            ->connections()
-            ->withHeader(new ContentType('application/json'))
+        if (empty($data['name'])) {
+            throw new \Exception('Missing required "name" field.');
+        }
+
+        if (empty($data['strategy'])) {
+            throw new \Exception('Missing required "strategy" field.');
+        }
+
+        return $this->apiClient->method('post')
+            ->addPath('connections')
             ->withBody(json_encode($data))
             ->call();
     }
 
     /**
-     * @param string $id
-     * @param array $data
-     * @return mixed
+     * Update a Connection.
+     *
+     * @param string $id - Connection ID to update.
+     * @param array $data - Connection data to update.
+     *
+     * @return mixed|string
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Connections/patch_connections_by_id
      */
-    public function update($id, $data) 
+    public function update($id, $data)
     {
-        return $this->apiClient->patch()
-            ->connections($id)
-            ->withHeader(new ContentType('application/json'))
```

Comment:
This one is now missing. Was missing from #256 too. Are you covering this somewhere else?

Reply:
`method('patch')` adds that header automatically